app/models/profile.rb

```diff
@@ -74,12 +76,13 @@ def canonical?
     parent_profile_id.blank?
   end
 
-  def clone_to(account:, policy:)
+  def clone_to(account:, policy:, os_minor_version: '')
     new_profile = in_account(account, policy)
     if new_profile.nil?
       (new_profile = dup).update!(account: account,
                                   parent_profile: self,
                                   external: true,
+                                  os_minor_version: os_minor_version,
                                   policy: policy)
       new_profile.update_rules(ref_ids: rules.pluck(:ref_id))
     end
```

Comment:
This is one way we could make `clone_to` smarter about when to add an `os_minor_version`. Now the caller doesn't have to know about any of this logic, and it's encapsulated here in the profile model.

```suggestion
  def clone_to(account:, policy:, os_minor_version: nil)
    new_profile = in_account(account, policy)
    if new_profile.nil?
      (new_profile = dup).update!(account: account,
                                  parent_profile: self,
                                  external: true,
                                  os_minor_version: os_minor_version,
                                  policy: policy)
      new_profile.update_rules(ref_ids: rules.pluck(:ref_id))
    end
    new_profile.update!(os_minor_version: os_minor_version) if os_minor_version && new_profile.os_minor_version.empty?
```